CMakeLists.txt

```diff
@@ -186,13 +207,15 @@ elseif ((CMAKE_Fortran_COMPILER_ID STREQUAL "NVHPC") OR (CMAKE_Fortran_COMPILER_
     if (CMAKE_BUILD_TYPE STREQUAL "Debug")
         add_compile_options(
             $<$<COMPILE_LANGUAGE:Fortran>:-O0>
+            -C 
```

Comment:
i'm not sure this syntax works. you could double check by looking at the cmake output of a specific run. i think you need to add
`  $<$<COMPILE_LANGUAGE:Fortran>:-<flag>>` on separate lines.

Reply:
I will fix this in my next commit. Thanks